pkg/ios/dsym-utils.go

```diff
@@ -133,7 +149,8 @@ func findDsyms(root string) []string {
 			return err
 		}
 
-		if strings.HasSuffix(strings.ToLower(info.Name()), ".dsym") {
+		// If the file is a dSYM, add it to the list (unless it resides within the __MACOSX directory)
+		if strings.HasSuffix(strings.ToLower(info.Name()), ".dsym") && !strings.Contains(strings.ToLower(path), "__macosx") {
```

Comment:
I changed this because we used to do the same thing in bugsnag-dsym-upload ([see here](https://github.com/bugsnag/bugsnag-dsym-upload/blob/47fee06af3a6d92e8aafd3c9d3453649ded03e13/bin/bugsnag-dsym-upload#L136)).